src/pages/Fox/components/RFOXSection.tsx

```diff
@@ -0,0 +1,301 @@
+
+  const stakingBalanceCryptoPrecision = useMemo(() => {
+    if (!(stakingBalanceCryptoBaseUnitResult.data && stakingAsset)) return
+    return fromBaseUnit(stakingBalanceCryptoBaseUnitResult.data, stakingAsset.precision)
+  }, [stakingAsset, stakingBalanceCryptoBaseUnitResult.data])
+
+  const currentEpochMetadataResult = useCurrentEpochMetadataQuery()
+
+  const currentEpochRewardsCryptoBaseUnitResult = useCurrentEpochRewardsQuery({
+    stakingAssetAccountAddress,
+    currentEpochMetadata: currentEpochMetadataResult.data,
+  })
+
+  const lifetimeRewardsCryptoBaseUnitResult = useLifetimeRewardsQuery({
+    stakingAssetAccountAddress,
+  })
+
+  const timeInPoolHumanResult = useTimeInPoolQuery({
+    stakingAssetAccountAddress,
+    select: timeInPoolSeconds =>
+      timeInPoolSeconds === 0n ? 'N/A' : formatSecondsToDuration(Number(timeInPoolSeconds)),
+  })
+
+  const affiliateRevenueResult = useAffiliateRevenueQuery<string>({
+    startTimestamp: currentEpochMetadataResult.data?.epochStartTimestamp,
+    endTimestamp: currentEpochMetadataResult.data?.epochEndTimestamp,
+    select: (totalRevenue: bigint) => {
+      return bn(fromBaseUnit(totalRevenue.toString(), runeAsset?.precision ?? 0))
+        .times(runeAssetMarketData.price)
+        .toFixed(2)
+    },
+  })
+
+  const burn = useMemo(() => {
+    if (!affiliateRevenueResult.data) return
+    if (!currentEpochMetadataResult.data) return
+
+    return bn(affiliateRevenueResult.data)
+      .times(currentEpochMetadataResult.data.burnRate)
+      .toFixed(2)
+  }, [affiliateRevenueResult.data, currentEpochMetadataResult.data])
+
+  if (!stakingAsset) return null
+
+  if (!isRFOXEnabled) return null
+
+  return (
+    <>
+      <Divider mt={2} mb={6} />
+      <Box py={4} px={containerPaddingX}>
+        <Flex sx={headerSx}>
+          <Box mb={headerTitleMb}>
+            <Heading as='h2' fontSize='2xl' display='flex' alignItems='center'>
+              <RFOXIcon me={2} boxSize='32px' sx={rfoxIconStyles} />
+              {translate('foxPage.rfox.title')}
+              <Tag colorScheme='green' verticalAlign='middle' ml={2}>
+                <Amount.Percent value={apyQuery.data ?? 0} suffix='APY' />
+              </Tag>
+            </Heading>
+            <Text fontSize='md' color='text.subtle' mt={2} translation='foxPage.rfox.whatIs' />
+          </Box>
+
+          <Card>
+            <CardBody py={4} px={4}>
+              <Text fontSize='md' color='text.subtle' translation='RFOX.pendingRewardsBalance' />
+
+              <Skeleton
+                isLoaded={
+                  !Boolean(
+                    currentEpochRewardsCryptoBaseUnitResult.isLoading ||
+                      currentEpochRewardsCryptoBaseUnitResult.isFetching,
+                  )
+                }
+              >
+                <Amount.Crypto
+                  value={fromBaseUnit(
+                    currentEpochRewardsCryptoBaseUnitResult.data?.toString(),
```

Comment:
I used a useMemo in [ecfde77](https://github.com/shapeshift/web/pull/7910/commits/ecfde77a27a503630df47c7c5e6156a6c19772e8)

`useCurrentEpochMetadataQuery` is combining TXs, I would like to avoid touching this part in this PR 🙏

Reply:
Sounds safe to me!